src/univariate/locationscale.jl

```diff
@@ -139,9 +139,30 @@ gradlogpdf(d::ContinuousAffineDistribution, x::Real) = gradlogpdf(d.ρ,(x-d.μ)/
 
 #### Syntactic sugar for simple transforms of distributions, e.g., d + x, d - x, and so on
 
+"""
+    +(d::Distribution, μ)
+
+Return a distribution that has been shifted by `μ`. If ``X ~ d``, this returns the distribution of ``μ + X``.
+"""
```

Comment:
This is unrelated and should be part of an improvement of the docs. There are already some open PRs that discuss what would be desired.

```suggestion
```

(BTW I think mu seems to suggest incorrectly that it's the mean or location of the resulting distribution, I think eg `x` would be better.)